usage/app/lib/ContentPollStream.scala

```diff
@@ -107,8 +115,28 @@ trait ContentPollStream {
   }
 
   val pollInterval  = Duration(pollIntervalInSeconds, SECONDS)
-  val rawObservable =  Observable.interval(pollInterval)
-    .flatMap(_ => getContent)
+  case class PollAttempt(lastFail: Throwable, i: Int)
+
+  lazy val retryingPollObservable = Observable.interval(pollInterval)
+    .flatMap(_ => getContent).retryWhen(
+      _.zipWith(Observable.from(1 to maxRetries))((e: Throwable, i: Int) => PollAttempt(e,i))
+      .flatMap(attempt => {
+        val isLastAttempt = attempt.i >= maxRetries
+        val waitSeconds = pow((attempt.i),2).second
+
+        val logString = s"Request to CAPI failed (attempt ${attempt.i} of ${maxRetries})!\n"
+        val retryMessage = s"${attempt.lastFail.getMessage}\nTrying again in ${waitSeconds}"
+        val fullMessage = if(isLastAttempt) logString else logString + retryMessage
+
+        Logger.info(fullMessage)
+
+        if (isLastAttempt) throw attempt.lastFail
+        Observable.timer(waitSeconds)
+      })
+    )
+
+  lazy val rawObservable = retryingPollObservable
+    .observeOn(scheduler)
```

Comment:
If we do not use a single threaded scheduler here it will simply try the next poll event on a different thread!